en/sql/schema/index_stmt.rst

```diff
@@ -327,7 +327,7 @@ The following is a syntax of rebuilding an index.
 
 ::
 
-    ALTER INDEX index_name ON [schema_name.]table_name REBUILD;
+    ALTER INDEX index_name ON [schema_name.]table_name {{[COMMENT index_comment_string] REBUILD} | {COMMENT index_comment_string} | VISIBLE | INVISIBLE};
 
 *   *index_name*: specifies the name of the index to be recreated. The index name must be unique in the table.
```

Comment:
not only "recreated" as REBUILD is applied in this ALTER clause but also can be applied as COMMENT, VISIBLE, or INVISIBLE.
I recommend "altered or recreated" instead of "recreated"

Reply:
OK, thank you.